utils/string.ts

```diff
@@ -2,4 +2,4 @@ export const isEns = (x: string) => x.endsWith('.eth');
 
 export const hasNonAsciiCharacters = (str: string) => /[^\u0000-\u007f]/.test(str);
 
-export const removeZeroes = (str: string) => str.replace(/(.0|(?<=(..))0*)$/, '');
+export const removeZeroes = (str: string) => str.replace(/(\.[0-9]*[1-9])0+$|\.0*$/, '$1');
```

Comment:
This is a better regex expression to remove zeroes, I have found some issue with the previous one.

Reply:
do we still want to change this now? we have everything working in prod at the moment. I dont want to break anything